src/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { TypographyVariantEnum } from '~/types'
+export const styles = {
+  topTitle: {
+    mt: 1
```

Comment:
here you set "1" for mt and below you set "40px", better to use one of these options everywhere. We prefer to use px

Reply:
Got it